spec/fixtures/babel/single-quotes.js

```diff
@@ -1,3 +1,3 @@
-'use 6to5';
+'use babel';
```

Comment:
Instead of changing the specs, I'd prefer to just add new ones for babel, so we make sure the legacy support is still tested.